### DISCUSSION TITLE
convert me to an issue

### DISCUSSION THREAD

**Discussion by QRLmining** [Merkletree-Testing/Discussions#23] [General]
this should be an issue but i started as a discussion

**Comment by jackalyst** [comment]
This is an actual issue, thanks for bringing it to our attention. Issue can be tracked over in our wallet repo
https://github.com/Merkletree-Testing/public_repository_demo2/issues/3

**Comment by fr1t2** [comment]
This is also referenced in this issue #21

**Comment by jackalyst** [comment]
public_repository_demo2#3

**Comment by fr1t2** [comment]
Closes #21

**Comment by fr1t2** [comment]
this is closed

**Comment by jackalyst** [comment]
Please move discussions to the issue
